daprdocs/content/en/developing-applications/building-blocks/bindings/howto-bindings.md

```diff
@@ -106,7 +106,7 @@ spec:
 
 The code examples below leverage Dapr SDKs to invoke the output bindings endpoint on a running Dapr instance. 
 
-{{< tabs Dotnet Java Python Go JavaScript>}}
+{{< tabs .NET Java Python Go JavaScript>}}
```

Comment:
```suggestion
{{< tabs ".NET" Java Python Go JavaScript>}}
```

I believe .NET needs to be in quotes in this shortcode - this may fix the PR build error.